src/generators/java/jsoup.ts

```diff
@@ -0,0 +1,120 @@
+import * as util from "../../util.js";
+import type { Request, Warnings } from "../../util.js";
+
+import jsesc from "jsesc";
+
+const supportedArgs = new Set([
+  "url",
+  "request",
+  "user-agent",
+  "cookie",
+  "data",
+  "data-raw",
+  "data-ascii",
+  "data-binary",
+  "data-urlencode",
+  "json",
+  "referer",
+  "form",
+  "form-string",
```

Comment:
```suggestion
  // "form",
  // "form-string",
```

Reply:
Fixed

Reply:
Supporting these arguments means reading `request.multipartUploads` and generating code that sends `multipart/form-data` correctly, for example like this

https://github.com/curlconverter/curlconverter/blob/bca093c8a86802427f0dd5a3f0a11fa7a6db2f5b/src/generators/csharp.ts#L187

Reply:
Thanks for comment 👍  Fixed.